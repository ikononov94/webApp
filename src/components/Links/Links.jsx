import React from 'react';
import { Link } from 'react-router-dom';
import { Button } from 'react-bootstrap';

const Links = () => (
  <div>
    <Link to="/">
      <Button bsStyle="link">Сетка</Button>
    </Link>
    <Link to="/departments">
      <Button bsStyle="link">Отделы</Button>
    </Link>
    <Link to="/employees">
      <Button bsStyle="link">Сотрудники</Button>
    </Link>
  </div>
);

export default Links;
