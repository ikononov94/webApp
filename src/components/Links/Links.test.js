import React from 'react'
import { shallow } from 'enzyme';
import Links from './Links';
import { Button } from 'react-bootstrap';

test('should render <Links />', () => {
    const wrapper = shallow(<Links />);
    
    expect(wrapper.find(Button).length === 3).toEqual(true);
    expect(wrapper).toMatchSnapshot();
});
