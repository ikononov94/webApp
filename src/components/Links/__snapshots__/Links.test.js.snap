// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <Links /> 1`] = `
<div>
  <Link
    replace={false}
    to="/"
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="link"
      disabled={false}
    >
      Сетка
    </Button>
  </Link>
  <Link
    replace={false}
    to="/departments"
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="link"
      disabled={false}
    >
      Отделы
    </Button>
  </Link>
  <Link
    replace={false}
    to="/employees"
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="link"
      disabled={false}
    >
      Сотрудники
    </Button>
  </Link>
</div>
`;
